net.labymod.LabyMod.metainfo.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>net.labymod.LabyMod</id>
+  
+  <name>LabyMod</name>
+  <summary>All-in-one Minecraft Client</summary>
+  
+  <metadata_license>MIT</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
```

Comment:
Is redistribution explicitly allowed?

Reply:
If your point is that I am not allowed to publish the app on Flatpak - I have obtained the CEO's permission to do so.

Reply:
it was a question.

Reply:
I see. As far as I know redistribution is not explicitly allowed.

Reply:
so it should use `extra-data`

https://docs.flathub.org/docs/for-app-authors/requirements#license

Reply:
Is that really necessary? I mean, I have permission to redistribute the program for this purpose. 
I didn't expect this to take so long and I'm just running out of time for this project. 
If I absolutely have to, I'll have to take the time somehow, but I'm afraid that familiarizing myself with the extra data system will cost me at least a few more hours :/

Reply:
All right, I have updated the installation method. I hope it's ok that I have now reinstalled the .desktop file, I didn't know how to export it from the extra-data.